README.md

```diff
@@ -44,7 +43,7 @@ FLAGS:
 
 OPTIONS:
     -p, --base-path <base-path>    Set path for chain storage
-    -c, --chain <chain>            Sets a chain name [default: mainnet]
+    -c, --chain <chain>            Sets a chain name [default: etc]
```

Comment:
why?

Reply:
Because ETH and ETC can't share same name

Reply:
but ETH is ETH, not mainnet, it's different name. I mean it's ok to have etc as a synonym to mainnet, but it shouldn't eliminate mainnet, because it's what expected to use for mainnet

Reply:
ok, I see. Will make then prefix for specific blockchain (`etc-..`|`eth-...`) whenever using specific chain, and leave as is for distinctive name.